Civi/Test/CiviCodeceptionTrait.php

```diff
@@ -0,0 +1,125 @@
+<?php
+namespace Civi\Test;
+
+/**
+ * Trait CiviCodeceptionTrait
+ * Trait for common functions used for Codeception End-to-End Testing framework
+ * @package Civi\Test
+ */
+trait CiviCodeceptionTrait {
```

Comment:
We discussed this a little face-to-face -- it'd be better to rename this `\Civi\Test\Codeception\CiviAcceptanceTesterTrait`. This naming convention makes it easier to differentiate the helper functions/implementations for acceptance/functional/unit tests.